tests/docker/Dockerfile

```diff
@@ -64,6 +64,10 @@ RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFK
     make install && \
     ldconfig
 
+# install dependencies for kafka clients
+RUN mkdir -p /opt/operatrio
+COPY tests/rptest/tests/compatibility/operatrio.py /opt/operatrio/operatrio.py
```

Comment:
@ivotron 
Is there a preferred way to bring example scripts or drivers into the containers? I'll probably end up creating my own repo with all the kafka client drivers so there is a central location.

Reply:
we can add code to ducktape itself no?

Reply:
The particular driver in this PR could be within ducktape, however, drivers from other kafka clients cannot. For example, [sarama drivers](https://github.com/Shopify/sarama/tree/master/examples) which are written in Golang and must be downloaded.

My thought process was to put all drivers into a single repo (or something like that) instead of some drivers within ducktape and others not within ducktape.

Reply:
the "guideline" we follow is:
  * if it's a git repo, we fork it to vectorizedio/ and `git clone` from the fork
  * if it's a pre-compiled binary, we put it in s3, and fetch via `curl` from the bucket.
  * if it's source code distributed as tarball, we also put it in s3.
  * if it's a script/program that we authored (or a modified/extended version of a 3rd-party one), it can live in the `tests/` folder as Alex mentioned.
  
would this help?